models/models.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class meeting__scheduler(models.Model):
+    _name = 'meeting__scheduler.meeting__scheduler'
```

Comment:
@NilsNeeb You can call the object e.g. "meeting.scheduler" that's totally fine, then you don't have that repetition